man/systemd.network.xml

```diff
@@ -229,6 +229,17 @@
           if <literal>RequiredForOnline=no</literal>.</para>
         </listitem>
       </varlistentry>
+      <varlistentry>
+        <term><varname>RequiredFamilyForOnline=</varname></term>
+        <listitem>
+          <para>Specifies an address family, which must be reached by the link in addition to the range of operational states.
```

Comment:
Same here, `an address family, which must be reached` sounds slightly strange for me...